build.gradle.kts

```diff
@@ -60,6 +60,7 @@ intellij {
     updateSinceUntilBuild.set(true)
     instrumentCode.set(true)
     plugins.set(listOf("yaml", "java", "IdeaVIM:0.69"))
+    localPath.set(property("localPath").toString())
```

Comment:
What's that?

Reply:
This is a mistake, good catch. I'm deleting it